cmd/prometheus/main.go

```diff
@@ -421,6 +422,9 @@ func main() {
 	serverOnlyFlag(a, "rules.max-concurrent-evals", "Global concurrency limit for independent rules that can run concurrently.").
 		Default("4").Int64Var(&cfg.maxConcurrentEvals)
 
+	serverOnlyFlag(a, "rules.evaluation-delay", "Duration to delay the evaluation of rules to ensure the underlying metrics have been received.").
```

Comment:
[nit] I suggest to clarify why there could be delays. Not sure my suggestion is clear enough tho.

```suggestion
	serverOnlyFlag(a, "rules.evaluation-delay", "Duration to delay the evaluation of rules to ensure the underlying metrics have been received with Prometheus is configured to receive metrics from a remote write endpoint.").
```

Reply:
Done in https://github.com/prometheus/prometheus/pull/14061/commits/164df4152bdb7f55fc4d6208eaa761dbaf409af3